libpod/options.go

```diff
@@ -809,3 +809,13 @@ func WithConmonPidFile(path string) CtrCreateOption {
 		return nil
 	}
 }
+
+func WithAdditionalGroups(groups []string) CtrCreateOption {
```

Comment:
Rename to `WithGroups` and add a comment for the function

Reply:
fixed.